packages/cambodia/README.md

```diff
@@ -0,0 +1,53 @@
+# 🇰🇭 Cambodia

```

Comment:
Could you add a link to this package in the main README please?

Reply:
Yes, I had added link to the Cambodia package in the main README.